content/master/cli/command-reference.md

```diff
@@ -320,6 +320,195 @@ flags, options or outputs in future releases.
 Crossplane maintainers may promote or remove commands under `beta` in future 
 releases.
 
+### beta validate
+
+The `crossplane beta validate` validates [compositions]({{<ref "../concepts/compositions">}}) against provider or XRD schema using Kubernetes API server's validation library.
+
+The validate command enables the following validation scenarios:
+
+- Validate any Crossplane resource against any schema (provider, XRD).
+- Use the output of the render command to validate function based compositions.
```

Comment:
```suggestion
- Use the output of the render command to validate function-based compositions.
```

Reply:
vale complains here `command-reference.md|330 col 52| Crossplane.Spelling:Spelling check: "function-based"?`

Reply:
The hyphen would be correct since this is a "a single adjective before a noun". Can you add `function-based` as an exception to 

utils/vale/styles/Crossplane/spelling-exceptions.txt